log4j-to-jul/src/main/java/org/apache/logging/log4j/tojul/JULLogger.java

```diff
@@ -0,0 +1,267 @@
+        LogRecord record = new LogRecord(julLevel, message.getFormattedMessage()); // NOT getFormat()
+        // NOT record.setParameters(message.getParameters()); BECAUSE getFormattedMessage() NOT getFormat()
+        record.setLoggerName(getName());
+        record.setThrown(t == null ? message.getThrowable() : t);
+        // Source class/method is not supported (yet)
+        record.setSourceClassName(null);
+        record.setSourceMethodName(null);
+        logger.log(record);
+        // fqcn is un-used
+    }
+
+    // Convert Level in Log4j scale to JUL scale.
+    // See getLevel() for the mapping. Note that JUL's FINEST & CONFIG are never returned because Log4j has no such levels, and
+    // that Log4j's FATAL is simply mapped to JUL's SEVERE as is Log4j's ERROR because JUL does not distinguish between ERROR and FATAL.
+    private java.util.logging.Level convertLevel(final Level level) {
+        switch (level.getStandardLevel()) {
+            // Test in logical order of likely frequency of use
+            // Must be kept in sync with #getLevel()
+            case ALL:
+                return java.util.logging.Level.ALL;
+            case TRACE:
+                return java.util.logging.Level.FINER;
+            case DEBUG:
+                return java.util.logging.Level.FINE;
+            case INFO:
+                return java.util.logging.Level.INFO;
+            case WARN:
+                return java.util.logging.Level.WARNING;
+            case ERROR:
+                return java.util.logging.Level.SEVERE;
+            case FATAL:
+                return java.util.logging.Level.SEVERE;
+            case OFF:
+                return java.util.logging.Level.OFF;
+            default:
+                // This is tempting: throw new IllegalStateException("Impossible Log4j Level encountered: " + level.intLevel());
+                // But it's not a great idea, security wise. If an attacker *SOMEHOW* managed to create a Log4j Level instance
+                // with an unexpected level (through JVM de-serialization, despite readResolve() { return Level.valueOf(this.name); },
+                // or whatever other means), then we would blow up in a very unexpected place and way. Let us therefore instead just
+                // return SEVERE for unexpected values, because that's more likely to be noticed than a FINER.
+                // Greetings, Michael Vorburger.ch <http://www.vorburger.ch>, for Google, on 2021.12.24.
+                return java.util.logging.Level.SEVERE;
+        }
+    }
+
+    /**
+     * Level in Log4j scale.
+     * JUL Levels are mapped as follows:
+     * <ul>
+     * <li>OFF => OFF
+     * <li>SEVERE => ERROR
+     * <li>WARNING => WARN
+     * <li>INFO => INFO
+     * <li>CONFIG => INFO
+     * <li>FINE => DEBUG
+     * <li>FINER => DEBUG (inspired by https://github.com/qos-ch/slf4j/blob/6e784e4ca3a5ae9c5dc421fcd01a417af5bf5ace/jul-to-slf4j/src/main/java/org/slf4j/bridge/SLF4JBridgeHandler.java)
```

Comment:
This doesn't agree with `convertLevel` above:

https://github.com/apache/logging-log4j2/blob/a58a06bf2365165ac5abdde931bb4ecd1adf0b3c/log4j-to-jul/src/main/java/org/apache/logging/log4j/tojul/JULLogger.java#L83-L84

Let's be consistent with the `log4j-jul` module https://github.com/apache/logging-log4j2/blob/a58a06bf2365165ac5abdde931bb4ecd1adf0b3c/log4j-jul/src/main/java/org/apache/logging/log4j/jul/DefaultLevelConverter.java#L55-L75

Reply:
Oh! Right. I'll change this (right now).

Reply:
@carterkozak Better now? Resolve if OK, our shout if I'm somehow mixing up which way we're mapping what here.